suiteData.js

```diff
@@ -32,6 +38,15 @@ class SuiteData {
                 suiteData = await this.getSuiteData(request.query);
                 break;
             case "POST":
+                let tests = request.body?.tests;
+                if (_.isString(tests)) {
+                    const testsDir = path.join(config.testsDir, sanitize(tests));
+                    if (await exists(testsDir)) {
+                        request.body.tests = (await listDir(testsDir)).filter(f => path.extname(f) === '.transcript').map(f => ({path: path.join(testsDir, f)}));
```

Comment:
`f` is better written as `fileName`.
use line breaks to make this more readable instead of meaningless variable names:
```
(await ...)
    .filter(...)
    .map(...);
```

Reply:
fixed